sdk/src/main/java/com/kin/ecosystem/marketplace/view/MarketplaceFragment.java

```diff
@@ -184,33 +184,33 @@ public void notifyEarnItemInserted(int index) {
 
 	@Override
 	public void notifySpendItemRemoved(int index) {
-		spendRecyclerAdapter.notifyItemRemoved(index);
+//		spendRecyclerAdapter.notifyItemRemoved(index);
```

Comment:
All those method that now aren't doing anything, is it ok or you forgot to remove the comments?
If it is ok then how do you get updated and/or get notified by all the events?

Reply:
will be one list